### ISSUE TITLE
Command to reload project config

### ISSUE THREAD

**Issue by Dimfred** [OscarCreator/rsync.nvim#83]
I am changing the project config quite often, and would like to have a command to reload the config.
I think this should happen by default.

**Comment by OscarCreator** [comment]
Would multiple hosts per config solve this? See #8

In toml there could be headings for different configs like:
```toml
[remote1]
remote_path = ...
...

[remote2]
remote_path = ...
...
```

And then there should be a command to switch between these or extra commands to for example sync up to both and down from one of them.

**Comment by Dimfred** [comment]
It's a actually more of a thing that i still sometimes modify the project config and have add a new host while nvim is open. In my PR I added this if I saw correct it's not an issie to reload the config on safe.

**Comment by OscarCreator** [comment]
This is now solved in #108
